docker-compose.yml

```diff
@@ -2,26 +2,23 @@
 version: "3.6"
 services:
   ojs:
-    privileged: true
     build:
-      context: .
+      context: https://github.com/arturluizbr/docker-pkp-ojs.git
```

Comment:
I've haven't seen this pattern before.  I'm worried that pointing context to a repo could cause issues with fluctuating  repo changes that have not been published to an image.  Should we just use a published image here @bibliotechy ?

Reply:
I think we should use a published image or at some point publish or own, as it will be easier to deploy as well. but maybe we proceed with this for the time being.